### ISSUE TITLE
Create a widget

### ISSUE THREAD

**Issue by nesimtunc** [rainlab-inc/cothings-ios#9]
- [ ] See the beacon supported rooms at top
 - [ ] List user's favorite rooms
   - [ ] User can change the counter through the screen

**Comment by ishansharma** [comment]
I'd be happy to work on this. Is there a design that I can reference for the widget or should I just try to follow existing conventions of the app?

**Comment by nesimtunc** [comment]
Hi @ishansharma 

Thank you for your interest! This feature would be great if you can help. Yes, we don't have a design for that but just like you said you may follow the existing conventions of the app.

We don't have a " to favorite a room or group" feature but I think we can add a ⭐️  button in the group section row, so that way they can favorite a group to be shown in the widget.

What do you think?

**Comment by ishansharma** [comment]
Yep, that sounds good. The widget can be named "CTthings Favorites" and this won't need an entire new configuration. I'll get started on it.

**Comment by nesimtunc** [comment]
**CoThings Favorties** I guess? or **CoThings Widget** as a folder name. Thanks! I'll change the status to "In Progress" then.

**Comment by ishansharma** [comment]
I didn't get much done on this. But I guess that was for good considering the new widgets announced at WWDC. I will get switch to using the iOS 14 widget APIs. Sounds good?

**Comment by nesimtunc** [comment]
🤔  ...  Yeah why not, since it's gonna release in fall, I guess it will be a cool feature. Thanks in advance.
